core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -183,16 +184,19 @@ public Boolean apply(Void input) {
 
   @Override
   public void start() throws Exception {
+    LOG.info("Starting Alluxio master @{}", mRpcAddress);
```

Comment:
Is this logging necessary, I think we are already handling it in `ProcessUtils` (though that might not work for unit tests)